test/e2e-new/e2e_suite.go

```diff
@@ -154,6 +163,25 @@ var _ = Describe("operator-sdk", func() {
 			}
 			err = tc.Make("bundle-build", "BUNDLE_IMG="+bundleImage)
 			Expect(err).Should(Succeed())
+
+			By("generating the operator package manifests")
+			var genPkgManCmd *exec.Cmd
+			Expect(tc.Make("manifests")).Should(Succeed())
+			genPkgManCmd = exec.Command(tc.BinaryName, "generate", "packagemanifests",
+				"--kustomize",
+				"--interactive=false")
+			_, err = tc.Run(genPkgManCmd)
+			Expect(err).Should(Succeed())
```

Comment:
Super pedantic nit but `Expect(err).Should(Succeed())` just sounds a little odd when I read it out loud.

From the godocs for `Suceed()` it's meant to be passed the function and not the error it seems. And a single error return function at that.
```
Succeed passes if actual is a nil error Succeed is intended to be used with functions that return a single error value. Instead of

    err := SomethingThatMightFail()

    Expect(err).ShouldNot(HaveOccurred())

You can write:

    Expect(SomethingThatMightFail()).Should(Succeed())
```

If we're getting the err value first then `Expect(err).ShouldNot(HaveOccurred())` reads better.

But non-blocking.

Reply:
I agree, these should be `Expect(err).NotTo(HaveOccurred())`, which is idiomatic.